packages/vscode-extension/lib/orientation.js

```diff
@@ -0,0 +1,159 @@
+const { Dimensions, Platform, UIManager, NativeEventEmitter } = require("react-native");
+const inspectorBridge = require("./inspector_bridge");
+
+// Used to supress warnings about UIManager not implementing below methods
+// when passed to NativeEventEmitter.
+UIManager.addListener = () => {};
+UIManager.removeListeners = () => {};
```

Comment:
Don't spend too much time on this one, it should be fine as is, but:

This is a bit risky, since we're overwriting properties of RN's exports.
I think I'd prefer we did this directly before the call that causes the warnings to be logged, and restore the original values directly after, like
```tsx
const originalAdd = UIManager.addListener;
const originalRemove = UIManager.removeListener;
UIManager.addListener = () => {};
UIManager.removeListeners = () => {};
/* ... do the stuff ... */
UIManager.addListener = originalAdd;
UIManager.removeListeners = originalRemove;
```
Or maybe even replace the `console.warn` method temporarily instead.